lib/Pico.php

```diff
@@ -1830,11 +1830,22 @@ protected function sortPages()
         if ($orderBy === 'meta') {
             // sort by arbitrary meta value
             $orderByMeta = $this->getConfig('pages_order_by_meta');
-            uasort($this->pages, function ($a, $b) use ($alphaSortClosure, $order, $orderByMeta) {
-                $aSortValue = isset($a['meta'][$orderByMeta]) ? $a['meta'][$orderByMeta] : null;
+            // Split the configuration value into an array of keys to allow sorting by nested meta values.
```

Comment:
The code is pretty self-explanatory, I don't think that we need inline code comments here (same with the comment below). However, we do need documentation in `config.yml.template` and `CHANGELOG.md`.